buildenv

```diff
@@ -23,3 +40,53 @@ zopen_get_version()
   # Rather than hardcoding the version, obtain the version by running the tool/library
   echo "1.0.0"
 }
+
+# This is called first
+zopen_init()
+{
+  # Check for python
+  if ! type python3 >/dev/null ; then
+    printError "need python3 venv with packages: httplib2 and six"
+  fi
+
+
+  #python3 -m venv curl_python
+  #. ./curl_python/bin/activate
+  #pip3 install impacket
+
+
+  # Ensure path is correct
+  desired_path="zopen/usr/local/zopen/depot_tools/depot_tools-main"
+
+  if  expr "${PATH}" : ".*${desired_path}.*" 1>/dev/null ; then
+    echo "Good. depot_tools is in path."
+  else
+    echo "Path to depot_tools missing.  Adding it to path.."
+    export PATH=${PATH}:${HOME}/${desired_path}
```

Comment:
You don't need this if you update:
https://github.com/ZOSOpenTools/depot_toolsport/blob/main/buildenv#L43
with:
```
  echo "export PATH=\${PWD}:\$PATH"
```
make sure the vars are escaped above ^
Also, I dont believe you need `${HOME}/.local/bin` since you're creating a virtual environment